src/lib/explorer/TokenIndexExplorer.svelte

```diff
@@ -72,45 +92,35 @@
 		 * @param settings Setting for the table: https://datatables.net/reference/type/DataTables.Settings
 		 */
 		ajax: async function (data, callback, settings) {
-			const params = {};
-
-			if (chainSlug) {
-				params.chain_slug = chainSlug;
+			const urlParams = new URLSearchParams(getAjaxParams(data));
+			const url = `${backendUrl}/tokens?${urlParams}`;
+			const response = await fetch(url);
+
+			if (response.ok) {
+				const result = await response.json();
+				callback({
+					recordsTotal: result.total,
+					recordsFiltered: result.total,
+					data: result
+				});
+			} else {
+				settings.oLanguage.sEmptyTable = await decodeAjaxError(response);
+				callback({
+					data: [],
+					recordsTotal: 0,
+					recordsFiltered: 0
+				});
 			}
-
-			// https://tradingstrategy.ai/api/explorer/#/Token/web_tokens
-			const encoded = new URLSearchParams({chain_slug: params.chain_slug});
-			const url = `${backendUrl}/tokens?${encoded}`;
-			const resp = await fetch(url);
-
-			if (!resp.ok) {
-				// Decode 422 invalid input parameter error from the server
-				// with JSON payload
-				let errorDetails;
-				try {
-					// GenericErrorModel in OpenAPI
-					errorDetails = await resp.json();
-					callback({ error: errorDetails.message });
-				} catch (e) {}
-
-				console.log('API error:', resp, 'error details:', errorDetails);
-				return;
-			}
-
-			const result = await resp.json();
-			console.log('Got token result', result);
-
-			result.recordsTotal = result.length;
-			result.recordsFiltered = result.length;
-			result.data = result;
-
-			callback(result);
 		}
 	};
 </script>
 
 <div class="tokens">
-	<Datatable {columns} {options} clickableRows={true} />
+	<Datatable
+	  columns={columns}
+	  options={options}
+    clickableRows={true}
+	/>
```

Comment:
Curious why you replaced the shorthand `{param}` syntax? (I happen to like the shorthand syntax 😄)

Reply:
shorthanded back